monitoring/etcd/etcd.go

```diff
@@ -204,6 +229,26 @@ func (e *Exporter) getMembers() (map[string]string, error) {
 	return membersMap, nil
 }
 
+// healthStatus determines status of etcd member
+func (e *Exporter) healthStatus() (healthy bool, err error) {
+	result := struct{ Health string }{}
+	nresult := struct{ Health bool }{}
+	resp, err := e.client.Get(e.client.Endpoint("health", ""), url.Values{})
+	if err != nil {
+		return false, trace.Wrap(err)
+	}
+
+	err = json.Unmarshal(resp.Bytes(), &result)
+	if err != nil {
+		err = json.Unmarshal(resp.Bytes(), &nresult)
+	}
+	if err != nil {
+		return false, trace.Wrap(err)
```

Comment:
Can you add `resp.Bytes()` to the error to be able to determine why unmarshaling failed?